impl/src/main/java/jakarta/faces/component/UIData.java

```diff
@@ -2371,8 +2371,8 @@ public WrapperEvent(UIComponent component, FacesEvent event, int rowIndex) {
         this.rowIndex = rowIndex;
     }
 
-    private FacesEvent event = null;
-    private int rowIndex = -1;
+    private final FacesEvent event;
+    private final int rowIndex;
```

Comment:
Why was `-1` default removed? Seems a breaking change.

Reply:
Bump. If you can revert this then I can merge this PR.

Reply:
Nevermind. The PR didn't show that it was in a `class WrapperEvent extends FacesEvent`. Approved.